src/main.js

```diff
@@ -565,6 +565,28 @@ function handleAppWebContentsCreated(dc, contents) {
         userAgent: popupUserAgent[process.platform],
       });
     }
+
+    if (Utils.isTrustedRemoteUrl(server.url, parsedURL)) {
+      popupWindow = new BrowserWindow({
+        backgroundColor: '#000',
+        show: true,
+        width: 800,
+        height: 600,
+        center: true,
+        alwaysOnTop: true,
+        webPreferences: {
+          nodeIntegration: false,
+          contextIsolation: true,
+        },
+      });
+      popupWindow.once('closed', () => {
+        popupWindow = null;
+      });
+
+      popupWindow.setMenuBarVisibility(false);
+
+      popupWindow.loadURL(url);
+    }
```

Comment:
It's unclear to me what the difference between this and the `Utils.isPluginUrl` case above is. Couldn't they be handled together? Couldn't `/trusted/` just be made into a plugin URL to avoid a lot of these changes altogether? There's probably some subtle thing I'm not noticing.

Reply:
Main difference is ```/plugins``` has ```parent: mainWindow``` while ```/trusted/``` doesn't.